src/app/content/highlights/components/HighlightsPopUp.spec.tsx

```diff
@@ -127,65 +128,19 @@ describe('MyHighlights button and PopUp', () => {
     expect(focus).toHaveBeenCalled();
   });
 
-  it('handles event listeners on mount and unmount for onEsc util', () => {
-    const focus = jest.fn();
-    const addEventListener = jest.fn();
-    const removeEventListener = jest.fn();
-    const createNodeMock = () => ({focus, addEventListener, removeEventListener});
-
-    const component = renderer.create(<TestContainer services={services} store={store}>
-          <HighlightsPopUp />
-    </TestContainer>, {createNodeMock});
-
-    const isHtmlElement = jest.spyOn(appGuards, 'isHtmlElement');
-
-    isHtmlElement.mockReturnValueOnce(true);
-
-    act(() => { store.dispatch(openMyHighlights()); });
-
-    expect(addEventListener).toHaveBeenCalled();
-
-    component.unmount();
-
-    expect(removeEventListener).toHaveBeenCalled();
-  });
-
-  it('handles event listeners on component update for onEsc util', () => {
-    const focus = jest.fn();
-    const addEventListener = jest.fn();
-    const removeEventListener = jest.fn();
-    const createNodeMock = () => ({focus, addEventListener, removeEventListener});
-
-    renderer.create(<TestContainer services={services} store={store}>
-      <HighlightsPopUp />
-    </TestContainer>, {createNodeMock});
-
-    const isHtmlElement = jest.spyOn(appGuards, 'isHtmlElement');
-
-    isHtmlElement.mockReturnValue(true);
-
-    act(() => { store.dispatch(openMyHighlights()); });
-
-    expect(addEventListener).toHaveBeenCalled();
-
-    // Force componentDidUpdate()
-    act(() => { store.dispatch(closeMyHighlights()); });
-
-    expect(removeEventListener).toHaveBeenCalled();
-  });
-
```

Comment:
I couldn't really figure out a way to mock these that worked with the new single handler... I'm sure there's a way but I feel like I spent enough hours on these tests...